tests/test_commands.py

```diff
@@ -49,6 +55,57 @@ def test_command__exit_code_2():
     assert log.error == test_command.EXCEPTION_MSG
 
 
+@pytest.mark.django_db
+class TestDisableModelSignalsOption:
+    @pytest.mark.parametrize("model_signals_disabled", (True, False))
+    @patch(f"tests.management.commands.test_command.Command.do_command")
+    def test_disables_model_signals_during_do_command(
```

Comment:
We're only testing that the `do_command` method is properly wrapped by the `disable_signals` context manager here. To test `--disable-model-signals` works by checking the signals themselves is much more complex since those signals already have side effects created by Django.